### PR TITLE
Revert DNS Settings

### LINKED ISSUE #157: dns article
changing to google dns

### PR DESCRIPTION
**Pull request by K97i**
closes #157

### CODE REVIEW

**Comment by Tomat0r** [comment]
[dns.md](https://github.com/r-Techsupport/rTS_Wiki/files/10579123/dns.md)

**Comment by K97i** [comment]
@Tomat0r the pull request shows the changes between the two repositories under the files changed tab so you dont have to upload the raw markdown file

image

**Comment by TheKrol** [comment]
Maybe update to link to terms?

**Review by K97i — COMMENTED** [review]
dns.md
add
`{: .no_toc}`
`{% include toc.md %}`
to AFTER `# Understanding DNS and How` and before `## What is DNS?`

instead of `6. Select "Use the following DNS server addresses" and enter the preferred DNS server addresses for the provider you want to use (e.g. Google's DNS servers are 8.8.8.8 and 8.8.4.4, Cloudflare's are 1.1.1.1 and 1.0.0.1, and OpenDNS's are 208.67.222.222 and 208.67.220.220).`

use `6. Select "Use the following DNS server addresses" and enter the preferred DNS server addresses for the provider [you want to use](#list-of-dns-providers)` replace all of them

**Review by TheKrol — CHANGES_REQUESTED** [review]
Made a PR to Tomator for DNS changes

**Review by WhiteHusky — CHANGES_REQUESTED** [review]
Various suggestions.

Also it might be a good idea to mention `/etc/crypttab` and how it's used to help facilitate mounting LUKS protected file systems at boot. Maybe even LVM since it's sometimes better to have one LUKS block device holding a LVM for multiple file systems.

File: docs/networking/dns.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+1. Click the Apple menu and select "System Preferences."
+2. Click "Network."
+3. Select the network connection you want to change and click "Advanced."
+4. Click the "DNS" tab.
+5. Click the "+" button to add a new DNS server address and enter the preferred DNS server addresses for the provider you want to use (e.g. Google's DNS servers are 8.8.8.8 and 8.8.4.4, Cloudflare's are 1.1.1.1 and 1.0.0.1, and OpenDNS's are 208.67.222.222 and 208.67.220.220).
+6. Click OK to save the changes.
+
+### Linux
+
+The specific steps to change the DNS server in Linux depend on the distribution you are using. For most distributions, you can change the DNS server by editing the `/etc/resolv.conf` file and adding the preferred DNS server addresses for the provider you want to use (e.g. Google's DNS servers are 8.8.8.8 and 8.8.4.4, Cloudflare's are 1.1.1.1 and 1.0.0.1, and OpenDNS's are 208.67.222.222 and 208.67.220.220).
```
[the remark this answers is not in the document]
Comment (WhiteHusky) [reply docs/networking/dns.md:52]: [`systemd-resolved`](https://wiki.archlinux.org/title/Systemd-networkd) can be configured by hand or via [`netplan`](https://netplan.readthedocs.io/en/latest/).

File: docs/safety-security/disk-encryption.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+## Finding Bitlocker Recovery Keys 
+
+Bitlocker recovery keys can be found in several places, depending on how Bitlocker was set up:
+
+- In the Microsoft account: If the recovery key was saved to the Microsoft account, it can be found by signing in to the account and viewing the security information. Follow [this guide](https://support.microsoft.com/en-us/windows/finding-your-bitlocker-recovery-key-in-windows-6b71ad27-0b89-ea08-f143-056f5ab347d6).
+- In the OneDrive account: If the recovery key was saved to the OneDrive account, it can be found by signing in to the account and searching for "Bitlocker."
+- On a USB drive: If the recovery key was saved to a USB drive, the USB drive must be inserted into the computer to access the encrypted data.
+- With the Bitlocker Recovery Console: If the recovery key was not saved to the Microsoft account or a USB drive, it can be found using the Bitlocker Recovery Console.
+
+### It is not possible to Decrypt BitLocker without them and we will not asisst in doing so.
```
[the remark this answers is not in the document]
Comment (sealsrock12) [reply docs/safety-security/disk-encryption.md:105]: I agree with you, headings should be for sections. You can change it to
```md
> **It is not possible to Decrypt BitLocker without them and we will not asisst in doing so.**
```
or 
```md
**It is not possible to Decrypt BitLocker without them and we will not asisst in doing so.**
```

[the remark this answers is not in the document]
Comment (Tomat0r) [reply docs/safety-security/disk-encryption.md:105]: Still figuring out how they work. Was aware * Exitsted but not the other things showed.

[the remark this answers is not in the document]
Comment (WhiteHusky) [reply docs/safety-security/disk-encryption.md:105]: Might be wise to use ❗ at the start, similar to what's being done on [Windows Maintenance](https://rtech.support/docs/recommendations/maintenance.html#defrag)

[the remark this answers is not in the document]
Comment (Tomat0r) [reply docs/safety-security/disk-encryption.md:105]: Will also move this to the top since it applies to every encryption method not only bitlocker

**Review by Tomat0r — APPROVED** [review]

**Review by Tomat0r — APPROVED** [review]

**Review by Tomat0r — APPROVED** [review]

File: docs/safety-security/disk-encryption.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+- Free and open-source
+- Cross-platform compatibility
+- Offers full disk encryption and encrypted volumes
+- Strong encryption options
+
+Disdvantages
+
+- May be more difficult to set up and use than other disk encryption software
+- Not as widely used as other disk encryption software, meaning there may be less support and resources available
```
[the remark this answers is not in the document]
Comment (WhiteHusky) [reply docs/safety-security/disk-encryption.md:160]: Though admittedly a footnote if anything. So perhaps the statement is still valid.

File: docs/networking/dns.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+1. Click the Apple menu and select "System Preferences."
+2. Click "Network."
+3. Select the network connection you want to change and click "Advanced."
+4. Click the "DNS" tab.
+5. Click the "+" button to add a new DNS server address and enter the preferred DNS server addresses for the provider [you want to use](#list-of-dns-providers).
+6. Click OK to save the changes.
+
+### Linux
+
+The specific steps to change the DNS server in Linux depend on the distribution you are using. For most distributions, you can change the DNS server by editing the `/etc/resolv.conf` file and adding the preferred DNS server addresses for the provider [you want to use](#list-of-dns-providers).
```
Comment (WhiteHusky) [inline docs/networking/dns.md:54]: Regarding [what was asked in the development channel](https://discord.com/channels/749314018837135390/1071069467955703888/1071760874135629835), it's a hard answer to ask on what's the most terse way to explain how to change the DNS that covers all the distros. We probably want people to stay within the GUI, so we could mention the few places where to find the settings in a couple common DEs (KDE, Gnome, Mate are a few that come to mind) and as a last resort suggest finding what [network manager](https://wiki.archlinux.org/index.php?title=Network_configuration&oldid=766537#Network_managers) their system is using then use that to modify DNS.

**Review by WhiteHusky — CHANGES_REQUESTED** [review]

File: docs/networking/dns.md (hunk 4 above)
Comment (WhiteHusky) [reply docs/networking/dns.md:54]: Though for desktop geared distros, it's almost always going to be `NetworkManager` that handles networking.

**Review by Tomat0r — APPROVED** [review]

**Review by sealsrock12 — APPROVED** [review]

**Pull request merged**